lib/scala/runtime-version-manager/src/main/scala/org/enso/runtimeversionmanager/runner/RunSettings.scala

```diff
@@ -8,9 +8,11 @@ import org.enso.semver.SemVer
   * @param runnerArguments arguments that should be passed to the runner
   * @param connectLoggerIfAvailable specifies if the ran component should
   *                                 connect to launcher's logging service
+  * @param attachDebugger if true, indicates that the runner should be started with remote debugging capabilities
```

Comment:
Let's make the docs a little bit more precise, pls. Starting something with debugging capabilities is very vague. Replace that with something like:
```suggestion
  * @param attachDebugger if true, indicates that the runner should be suspended until a debugger is attached.
```

Maybe even change the param name to `waitForDebugger`?